test/runtests.jl

```diff
@@ -1,23 +1,27 @@
 using IntervalArithmetic
 using Test
 
+IntervalArithmetic.:(≛)(a::Tuple, b::Tuple) = all(a .≛ b)
 
-const Interval = IntervalArithmetic.Interval
-
-# Interval tests:
+function include_test(filename)
+    @testset "$filename" begin
+        include(filename)
+    end
+end
 
 setformat(:full)
 
-include("interval_tests/intervals.jl")
-include("decoration_tests/decoration_tests.jl")
+# Interval tests:
+include_test("interval_tests/intervals.jl")
+include_test("decoration_tests/decoration_tests.jl")
 
-include("rand.jl")
+include_test("rand.jl")
 
 # Display tests:
-include("display_tests/display.jl")
+include_test("display_tests/display.jl")
 
 # ITF1788 tests
 
-include("ITF1788_tests/ITF1788_tests.jl")
+# include_test("ITF1788_tests/ITF1788_tests.jl")  # TODO fix these tests
```

Comment:
if/when they are re-included, I think it would make sense to have all of them clustered in one folder as they were before. (I am not sure now if they have been just temporarily removed or just scattered around)